### PR TITLE
feat: migrate upgrade of v7-board from  upgrade 19 to upgrade 18

### LINKED ISSUE #10760: Include upgrade of v7-board in upgrade 18
## What is the Problem Being Solved?

As noted in #10743 and elsewhere, the V7-board vat (and others) has been growing recently. We now believe we have a fix, but it requires upgrading the vat. Please include the upgrade of the board vat (#10394, #10516) in upgrade 18.

## Description of the Design

Uncomment the proposal for the board in `upgrade.go`, and clean up tests in `a3p-integration` to reflect the fact of the move.

## Security Considerations

None.

## Scaling Considerations

Addresses a memory leak.

## Test Plan

Tests are included in `p:upgrade-19`. Move them to `n:upgrade-next`.

## Upgrade Considerations

That's what it's about.

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
closes: #10760

## Description

Move the upgrade of the V7-board vat from upgrade-19 to upgrade-18.

### Security Considerations

None

### Scaling Considerations

Addresses a performance issue.

### Documentation Considerations

None

### Testing Considerations

There is an existing test. It moves along with the proposal.

### Upgrade Considerations

Including this upgrade in U18 is considered crucial for chain health.

### CODE REVIEW

**Review by dckc — APPROVED** [review]
looks about right.

File: a3p-integration/proposals/p:upgrade-19/package.json [hunk 1]
```diff
@@ -7,7 +7,6 @@
       "vats/upgrade-provisionPool.js upgradeProvisionPool",
       "vats/upgrade-asset-reserve.js upgradeAssetReserve",
       "vats/upgrade-paRegistry.js",
-      "vats/upgrade-board.js",
       "testing/test-upgraded-board.js testUpgradedBoard",
```
Comment (mhofman) [inline a3p-integration/proposals/p:upgrade-19/package.json]: I assume this should be deleted too?

```suggestion
```

File: a3p-integration/proposals/n:upgrade-next/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+testUpgradedBoard/
```
Comment (mhofman) [inline a3p-integration/proposals/n:upgrade-next/.gitignore:1]: Should we remove the equivalent line from https://github.com/Agoric/agoric-sdk/blob/75ca89e583ebfc05ca00fd215c0d78a055087162/a3p-integration/proposals/p%3Aupgrade-19/.gitignore#L2 ?

**Review by mhofman — APPROVED** [review]
Approved assuming we fully move the test bits from u19

File: a3p-integration/proposals/n:upgrade-next/.gitignore (hunk 2 above)
Comment (Chris-Hibbert) [reply a3p-integration/proposals/n:upgrade-next/.gitignore:1]: good idea. I'll add that once CI passes. (I don't want to restart the cycle at this point.)

File: a3p-integration/proposals/p:upgrade-19/package.json (hunk 1 above)
Comment (Chris-Hibbert) [reply a3p-integration/proposals/p:upgrade-19/package.json]: Yep. I'll get rid of that. I thought I had done so, but it's still there.

**Pull request merged**